website/pages/docs/configuration/telemetry.mdx

```diff
@@ -37,6 +37,22 @@ The following options are available on all telemetry configurations.
 - `enable_hostname_label` `(bool: false)` - Specifies if all metric values should
   contain the `host` label with the local hostname. It is recommended to enable
   `disable_hostname` if this option is used.
+- `lease_metrics_epsilon` `(string: "1hr")` - Specifies the time difference between collections
+  of leases by expiry time. For example, for the default value of 1 hour, the `vault.expire.leases.by_expiration`
+  metric will aggregate the total number of expiring leases for 1 hour buckets, starting from the current time.
+  Note that leases are put into buckets by rounding. For example, if `lease_metrics_epsilon` is set to 1hr and 
+  lease A expires 25 minutes from now, and lease B expires 35 minutes from now, then lease A will be in a
+  different bucket from lease B. 
+- `num_lease_metrics_buckets` `(int: 168)` - The number of expiry buckets for leases. For the default value for
+  example, 168 value labels for the `vault.expire.leases.by_expiration` metric will be reported, where each value 
+  has a time difference from the previous value that is specified by the `lease_metrics_epsilon` parameter. For the
+  default 1 hour value of `lease_metrics_epsilon` and the default value of `num_lease_metrics_buckets`, `vault.expire.leases.by_expiration`
+  will report the total number of leases expiring within each hour from the current time to one week from the current time.
+- `add_lease_metrics_namespace_labels` `(bool: false)` - If this value is set to true, then `vault.expire.leases.by_expiration` will break down
+  expiring leases by both time and namespace. For example, if x labels are expiring within one time bucket T specified by `lease_metrics_epsilon`
+  and 40% of those leases are in namespace a and 60% of those leases are in namespace b, then having this value set to true will yield two
+  different label sets -- one with value .4x with labels T and a, and another with value .6x and labels T and b.
```

Comment:
Sorry, I found the switch from 40% to .4x confusing and had to re-read.

I think the example is probably not necessary; what I'd prefer to say is that this is disabled by default because enabling it can lead to a large-cardinality metric.